### ISSUE TITLE
minLevel set

### ISSUE THREAD

**Issue by augusteo** [justeat/JustLog#3]
Hi, I wanted to set minLevel for console logging but the vars are private

```Swift
    private let internalLogger = SwiftyBeaver.self
    private var console: ConsoleDestination!
```

Do you have another way to set it or should I send a PR with a functionality to enable minLevel changes?

use case: I only wanted to see logs on warning and error, ignoring all the others.

**Comment by albertodebortoli** [comment]
@augusteo I think it's a valid point. It should be now fixed with #6 and will hopefully be merged in soon.

**Comment by federicocappelli** [comment]
Merged in master

**Comment by albertodebortoli** [comment]
This is now publicly available in trunk (v 1.1.0).
